README.md

```diff
@@ -52,41 +52,44 @@ For more information, you can refer to [ChainerRL's documentation](http://chaine
 | TD3 | x | ✓ | x | x |
 
 Following algorithms have been implemented in ChainerRL:
-- A3C (Asynchronous Advantage Actor-Critic)
-- ACER (Actor-Critic with Experience Replay)
-- Asynchronous N-step Q-learning
-- Rainbow
-- Categorical DQN
-- IQN
-- DQN (including Double DQN, Persistent Advantage Learning (PAL), Double PAL, Dynamic Policy Programming (DPP))
-- DDPG (Deep Deterministic Policy Gradients) (including SVG(0))
-- PGT (Policy Gradient Theorem)
```

Comment:
We're not going to include PGT in the new README?

Reply:
I wrote why I removed it in the PR.